stellar-contract-env-common/src/symbol.rs

```diff
@@ -282,6 +282,46 @@ impl FromIterator<char> for Symbol {
     }
 }
 
+use crate::xdr::ScVal;
+
+#[cfg(feature = "std")]
+impl TryFrom<ScVal> for Symbol {
+    type Error = ();
```

Comment:
ConversionError?

Reply:
We'll clean it up in follow-up work. I'm not up to date on the best practices regarding error propagation in the host crate.

Reply:
Opened #260